### PR TITLE
My Code Suggestions

### PR DESCRIPTION
**Pull request by nkatsscd**

### CODE REVIEW

File: 7024Movies/DiscoverMovieSchema.json [hunk 1]
```diff
@@ -35,10 +35,13 @@
       "additionalProperties": false,
       "properties": {
         "popularity": {
-          "type": "number"
+          "type": "number",
+          "minimum": 0,
```
Comment (nkatsscd) [inline 7024Movies/DiscoverMovieSchema.json:39]: Based on the scope of the project, although it is sometimes unknown it is still best to give the minimum and maximum range for the popularity range. This help shelve out irregular values. 
https://www.w3.org/WAI/tutorials/forms/validation/

File: 7024Movies/Pages/DiscoverThriller.cshtml.cs [hunk 2]
```diff
@@ -28,21 +28,21 @@ public void OnGet()
                 JObject jsonObject = JObject.Parse(JsonString);
                 if (jsonObject.IsValid(schema))
                 {
-                    var discoverthriller = DiscoverM.DiscoverMovie.FromJson(JsonString);
-                    var thrillermovies = discoverthriller.Results;
-                    ViewData["Thrillers"] = thrillermovies;
+                    var DiscoverThriller = DiscoverM.DiscoverMovie.FromJson(JsonString);
+                    var ThrillerMovies = DiscoverThriller.Results;
+                    ViewData["Thrillers"] = ThrillerMovies;
```
Comment (nkatsscd) [inline 7024Movies/Pages/DiscoverThriller.cshtml.cs:33]: By this best practices article it talks about using Carmelcase when declaring variables.
https://dev.to/prahladyeri/underscores-camelcasing-and-pascalcasing-the-three-naming-conventions-every-programmer-should-be-aware-of-3aed

File: 7024Movies/Pages/Testing.cshtml.cs [hunk 3]
```diff
@@ -18,8 +18,8 @@ public void OnGet()
         {
             using (var webClient = new WebClient())
             {
-                string JsonString = webClient.DownloadString("https://api.themoviedb.org/3/search/movie?api_key=a852a3b3771672da86800503084b853b&language=en-US&query=star&page=1&include_adult=false");
-                var results = SearchResults.FromJson(JsonString);
+                string MovieReviews = webClient.DownloadString("https://api.themoviedb.org/3/search/movie?api_key=a852a3b3771672da86800503084b853b&language=en-US&query=star&page=1&include_adult=false");
+                var results = SearchResults.FromJson(MovieReviews);
```
Comment (nkatsscd) [inline 7024Movies/Pages/Testing.cshtml.cs:22]: By this best practice it talks about why we should use a more concise word in naming variables. It helps in putting an allocating everything together. It also helps in preventing technical errors.

https://dzone.com/articles/best-practices-variable-and

File: 7024Movies/Pages/DiscoverThriller.cshtml.cs [hunk 4]
```diff
@@ -52,11 +52,11 @@ public void OnGet()
                     {
                         //k_3re1w44s
                         //k_vxcfqztc
-                        string movieid = movie.ImdbId.ToString();
-                        uri = String.Format("https://imdb-api.com/en/API/Title/k_3re1w44s/{0}", movieid);
+                        string MovieId = movie.ImdbId.ToString();
+                        uri = String.Format("https://imdb-api.com/en/API/Title/k_3re1w44s/{0}", MovieId);
```
Comment (nkatsscd) [inline 7024Movies/Pages/DiscoverThriller.cshtml.cs:56]: By this best practices article it talks about using Carmelcase when declaring variables
https://dev.to/prahladyeri/underscores-camelcasing-and-pascalcasing-the-three-naming-conventions-every-programmer-should-be-aware-of-3aed

File: 7024Movies/Pages/DiscoverThriller.cshtml.cs [hunk 5]
```diff
@@ -52,11 +52,11 @@ public void OnGet()
...
                         string ImdbString = webClient.DownloadString(uri);
-                        var imdbrating = IMDB.MovieImdb.FromJson(ImdbString);
-                        thrillerimdb.Add(imdbrating);
+                        var ImdbRating = IMDB.MovieImdb.FromJson(ImdbString);
+                        thrillerimdb.Add(ImdbRating);
```
Comment (nkatsscd) [inline 7024Movies/Pages/DiscoverThriller.cshtml.cs:59]: By this best practices article it talks about using Carmelcase when declaring variables
https://dev.to/prahladyeri/underscores-camelcasing-and-pascalcasing-the-three-naming-conventions-every-programmer-should-be-aware-of-3aed

**Comment by nkatsscd** [comment]
•	Summarize the three technical changes you made, and why.
I first made a change to the movie review API string. It was just named Json initially string initially. I wanted to give it a more specific identity. This change was made on the Testing.cshtml.cs page.
I also Added some validations to the DiscoverMovieSchema.json. Due to the description of the project it seemed fit for certain validation to be applied.
I made changes to the variable name used to follow some best practices. This change was made to the DiscoverMovieSchema.cshtml.cs.

•	List three items you learned from the code review.
I learnt more about naming conventions. The best practices helped in learning why specific and concise variable names is necessary.
I learnt about how to use Carmel case to declare a variable.
I learnt a different method in integrating data with api.
•	Was the project available in GitHub on time?
It was available on time.
•	Does the project compile?
The project Compiles.
•	Does the project have adequate documentation?
It has adequate documentation

File: 7024Movies/DiscoverMovieSchema.json (hunk 1 above)
Comment (discospiff) [reply 7024Movies/DiscoverMovieSchema.json:39]: Good idea.  Reasonable values never hurt.  :)

File: 7024Movies/Pages/DiscoverThriller.cshtml.cs (hunk 2 above)
Comment (discospiff) [reply 7024Movies/Pages/DiscoverThriller.cshtml.cs:33]: Java tends to be lowerCamelCase, while C# leans toward UpperCamelCase.  Either way, those are an improvement over alllowercase.

File: 7024Movies/Pages/Testing.cshtml.cs (hunk 3 above)
Comment (discospiff) [reply 7024Movies/Pages/Testing.cshtml.cs:22]: Good catch!  "JsonString" is easy, but could be anything.  "MovieReviews" is more descriptive and readable.

**Comment by discospiff** [comment]
Several good suggestions in this branch.  I recommend merging these changes.

**Pull request merged**